pontoon/base/models.py

```diff
@@ -1079,6 +1079,12 @@ def has_suggestions(self, locale):
     def unchanged(self, locale):
         return self.with_status_counts(locale).filter(unchanged_count=F('expected_count'))
 
+    def authored_by(self, locale, email):
+        return self.filter(translation__locale=locale, translation__user__email=email)
+
+    def between_time_interval(self, locale, start, end):
```

Comment:
I think we could use range here: https://docs.djangoproject.com/en/dev/ref/models/querysets/#range